2200014/1.py

```diff
@@ -0,0 +1,15 @@
+#1 5431 민석이의 과제 체크하기
+
+
+t=int(input())
+
+for i in range(1,t+1):
+    n,k=map(int,input().split())
+    arr=[]
+    arr.append(list(map(int,input().split())))
+    print(f'#{i}',end=' ')
+    for i in range(1,n+1):
+        if i not in arr[0]:
+            
+            print(i,end=' ')
+    print()
```

Comment:
깔끔한 풀이네요👍 
```suggestion
    arr=list(map(int,input().split()))
    print(f'#{i}',end=' ')
    for i in range(1,n+1):
        if i not in arr:
            
            print(i,end=' ')
    print()
```
`arr`에 `append`하는 대신, 바로 리스트를 선언하는 방법도 좋을 것 같습니다! 😄